sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_daily_v1/query.sql

```diff
@@ -68,50 +74,166 @@ WITH base AS (
       'sync - repair_triggered'
     )
 ),
-windowed AS (
+flow_entrypoints AS (
+  SELECT
+    flow_id,
+    ARRAY_AGG(
+      -- if logic here so that we have an entry for
+      -- flow_id even if no entrypoint is found.
+      IF(entrypoint IS NOT NULL, STRUCT(`timestamp`, entrypoint), NULL) IGNORE NULLS
+      ORDER BY
+        `timestamp`
+      LIMIT
+        1
+    )[SAFE_OFFSET(0)] AS flow_entrypoint_info
+  FROM
+    fxa_events
+  WHERE
+    flow_id IS NOT NULL
+  GROUP BY
+    flow_id
+),
+user_service_flow_entrypoints AS (
+  SELECT
+    user_id,
+    service,
+    ARRAY_AGG(
+      STRUCT(
+        flow_id,
+        flow_entrypoint_info.`timestamp`,
+        flow_entrypoint_info.entrypoint
+      ) IGNORE NULLS
```

Comment:
A structed defined this way will never be `NULL`, so `IGNORE NULLS` won't do anything unless you do something like this:
```suggestion
      IF(
        flow_entrypoint_info.entrypoint IS NOT NULL,
        STRUCT(
          flow_id,
          flow_entrypoint_info.`timestamp`,
          flow_entrypoint_info.entrypoint
        ),
        NULL
      ) IGNORE NULLS
```

Reply:
or you could add `flow_id` to `flow_entrypoint_info` in `flow_entrypoints`, so that here you just have:
```suggestion
      flow_entrypoint_info IGNORE NULLS
```